.github/workflows/reusable-audit.yml

```diff
@@ -47,8 +52,17 @@ jobs:
           ref: ${{ matrix.ref }}
       - name: Install tooling
         uses: asdf-vm/actions/install@4f8f7939dd917fc656bb7c3575969a5988c28364 # v3.0.0
+      - name: Insert custom configuration
+        if: ${{ startsWith(matrix.ref, 'v') }}
+        env:
+          GRYPERC: ${{ vars.GRYPERC }}
+        run: echo "${GRYPERC}" | tee .grype.yml
       - name: Audit dependencies in container image
+        if: ${{ !startsWith(matrix.ref, 'v') }}
         run: make audit-vulnerabilities-image
+      - name: Audit dependencies in container image
+        if: ${{ startsWith(matrix.ref, 'v') }}
+        run: make audit-image
```

Comment:
Added this because the new command, introduced in #969, hasn't landed in [`v0`](https://github.com/ericcornelissen/js-regex-security-scanner/tree/v0) yet.